openshift-ci/Dockerfile.registry.build

```diff
@@ -11,8 +15,13 @@ ENV REG_URL=registry.svc.ci.openshift.org
 
 # replaces performance-addon-operator image with the one built by openshift ci
 RUN find /registry/performance-addon-operator-catalog/ -type f -exec sed -i "s|REPLACE_IMAGE|${REG_URL}/${OPENSHIFT_BUILD_NAMESPACE}/stable:performance-addon-operator|g" {} \; || :
+
 # Initialize the database
+USER root
```

Comment:
why is this needed now?

Reply:
Because the base image now runs with USER 1001, but we trying to put the `bundle.db` under the `/` and the user 1001 not allowed to write under it.

Reply:
ok, so another change caused bu the base image changes :\